app/src/main/java/com/github/libretube/ui/views/CustomExoPlayerView.kt

```diff
@@ -196,7 +196,7 @@ internal class CustomExoPlayerView(
     }
 
     private fun cancelHideControllerTask() {
-        handler.removeCallbacksAndMessages(HIDE_CONTROLLER_TOKEN)
+        handler?.removeCallbacksAndMessages(HIDE_CONTROLLER_TOKEN)
```

Comment:
App was randomly crashing due to `NullPointerException`. I noticed several times today.

Reply:
I actually fixed the same thing in my open PR :joy: